database/migrations/20220213062646_create-user-table.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+
+exports.up = function (knex) {
+    return knex.schema.createTable('user_types', function (table) {
```

Comment:
Make sure the name of the file is meaningful. Here the file name is `..._create-user-table.js` and the file is for creating `user_types` table.